src/sacn/private/opts.h

```diff
@@ -320,6 +320,79 @@
 #define SACN_DMX_MERGER_MAX_SOURCES_PER_MERGER SACN_RECEIVER_MAX_SOURCES_PER_UNIVERSE
 #endif
 
+/**
+ * @}
+ */
+
+/***************************** sACN Universe Discovery Options *****************************/
+
+/**
+ * @defgroup sacnopts_universe_discovery sACN Universe Discovery Options
+ * @ingroup sacnopts
+ *
+ * Configuration options for the @ref sacn_universe_discovery module.
+ * @{
+ */
+
+/**
+ * @brief The priority of the sACN universe discovery thread.
+ *
+ * This is usually only meaningful on real-time systems.
+ */
+#ifndef SACN_UNIVERSE_DISCOVERY_THREAD_PRIORITY
+#define SACN_UNIVERSE_DISCOVERY_THREAD_PRIORITY ETCPAL_THREAD_DEFAULT_PRIORITY
```

Comment:
Are we starting another thread for universe discovery? Would it not work/make sense to piggyback on one of the receiver threads?

Reply:
It may.  I can add a note in here for Christian to remove if he decides to do that in implementation.  Good catch.